src/public/publicAPIs.ts

```diff
@@ -317,3 +318,13 @@ export function navigateToTab(tabInstance: TabInstance, onComplete?: (status: bo
   GlobalVars.callbacks[messageId] = onComplete ? onComplete : getGenericOnCompleteHandler(errorMessage);
 }
 
+export class AppLoadEvent implements IAppLoadEvent {
+  public notifySuccess(): void {
+    ensureInitialized();
+    sendMessageRequest(GlobalVars.parentWindow, "onTabShow.success", [version]);
```

Comment:
NIT: I think it would be better if we called this `appLoadEvent.success` or `initialize.success`

Reply:
Same for the failure case below